sdk/translation/azure-ai-translation-document/tests/testcase.py

```diff
@@ -104,7 +99,7 @@ def _validate_doc_status(self, doc_details, target_language=None, **kwargs):
         ids = kwargs.pop("ids", None)
         # specific assertions
         assert doc_details.status in status
-        if target_language:
+        if self.is_live and target_language:
```

Comment:
Why did we add this here?

Reply:
This was discussed in our Teams chat  and this is what Krista had told: " we added some new global sanitizers to the repo and I'm wondering if this could be related to the issue in the screenshot. It seems like the
translated_to response property is being recorded as sanitized and then failing on playback.I think this is possibly the culprit: https://github.com/mccoyp/azure-sdk-for-python/blob/f1b82b18c24a8bf056889dcda6ccc66969351a0b/tools/azure-sdk-tools/devtools_testutils/proxy_startup.py#L313. the property comes across the wire as
to. This is probably the best way to go about it for now given we have more sanitizer changes coming. Hamshavathi Munibyraiah (MURPHY AND ASSOCIATES INC) if you just want to add an if self.is_live and target_language:  
to that line code, then we'll just verify this live instead of relying on playback for now"